docs/guides/chat/set_chat_variables.md

```
@@ -17,15 +19,15 @@ When configuring the system prompt for a chat model, variables play an important
 In RAGFlow, variables are closely linked with the system prompt. When you add a variable in the **Variable** section, include it in the system prompt. Conversely, when deleting a variable, ensure it is removed from the system prompt; otherwise, an error would occur.
 :::
 
-## Where to set variables
+## Where to Set Variables
 
 ![](https://raw.githubusercontent.com/infiniflow/ragflow-docs/main/images/chat_variables.jpg)
```

markdownlint-cli2 (0.23.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)